torch/csrc/jit/codegen/cuda/graph_fuser.cpp

```diff
@@ -63,11 +63,12 @@ Value* createConditionalConstant(Node* profile_ivalue) {
         static_cast<bool>(profile_ivalue->i(Symbol::attr("profiled_bool"))));
   } else {
     GRAPH_DEBUG("profile_ivalue: ", *profile_ivalue);
-    TORCH_INTERNAL_ASSERT(
-        false,
-        __func__,
-        " gets unidentified type: ",
-        profile_ivalue->ty(attr::profiled_type));
+    return nullptr;
```

Comment:
nitpick: maybe throw a warning here, this is still pretty surprising that we see un-profiled values.